config/repositories.yml

```diff
@@ -6,7 +6,7 @@ nla:
   location_html: |
     <div class="al-repository-street-address-address1">Parkes Place</div>
     <div class="al-repository-street-address-city_state_zip_country">Canberra ACT 2600, Australia</div>
-  thumbnail_url: 'https://assets.nla.gov.au/logos/nla/nla.svg'
+  thumbnail_url: '/finding-aids/assets/nla-stacked-logo-black.svg'
```

Comment:
What's happening with the assets domain, that seems like it will break lots of other apps and seems less than ideal since all the shared resources now need to be stored in individual apps.

Reply:
@scott-yeadon I haven't updated assets with new logos yet as I didn't want to break anything that might still be using old branding (and didn't want to create duplicate old/new logos). Might be something to migrate to.